flash_examples/finetuning/style_transfer.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+import flash
+from flash.data.utils import download_data
+from flash.utils.imports import _PYSTICHE_AVAILABLE
+from flash.vision.style_transfer import StyleTransfer, StyleTransferData
+
+if _PYSTICHE_AVAILABLE:
+    import pystiche.demo
+else:
+    print("Please, run `pip install pystiche`")
+    sys.exit(0)
+
+download_data("http://images.cocodataset.org/zips/train2014.zip", "data")
+
+data_module = StyleTransferData.from_folders(train_folder="data", batch_size=4)
+
+style_image = pystiche.demo.images()["paint"].read(size=256)
```

Comment:
Can we replace this by an image from internet, like 

![](https://www.kazoart.com/blog/wp-content/uploads/2017/03/van-gogh-la-nuit-etoilee-1.jpg).

style_image could actually be url to style_transfer.

Reply:
Sure, just tell me which image you want to use. Not sure about licensing though. The images I used are licensed very permissive.

Reply:
as soon we reference well should be fine. Consider also adding a link to here in the docs: https://paperswithcode.com/task/style-transfer. Papers with code will be reference soon also to submit results.

Reply:
> Consider also adding a link to here in the docs: https://paperswithcode.com/task/style-transfer. Papers with code will be reference soon also to submit results.

The link is doing image-optimization whereas we are implementing model-optimization here.

Reply:
@tchaton I need a reference image to check if the training works well with the default parameters.

Reply:
Oh I see. Makes sense.